intl/conf.py

```diff
@@ -1,78 +1,15 @@
-import os
-import sys
-import time
-import ablog
+# Obtain shared config values
+import os, sys
 
-# Configuration file for the Sphinx documentation builder.
+sys.path.append(os.path.abspath(".."))
+sys.path.append(os.path.abspath("../.."))
 
-# -- Project information
-project = "Nuitka the Python Compiler"
-copyright = "%s, Kay Hayen and Nuitka Contributors" % time.gmtime().tm_year
-author = "Kay Hayen"
-release = version = ""
-
-# -- General configuration
-
-extensions = [
```

Comment:
The list of extensions will be different, I would have a `+= ["ablog"] ` on the main site, and probably more.  Otherwise it creates (empty) feed files in output, that are ugly to have. Unless, of course, we commit to translated feeds in the future. But I wouldn't start it now.